examples/nlp/text_classification/config.yaml

```diff
@@ -0,0 +1,58 @@
+# Sentence classification with pretrained BERT models
+batch_size: 32
```

Comment:
Is it a good idea to put the help part of each param in front of it as a comment? It helps the user to understand them better.

Reply:
Yes, I agree we can comment YAML. There's also really nice things we can do with Hydra for help from the command line: https://hydra.cc/docs/next/configure_hydra/app_help

Reply:
Looks like a good idea! Lets see how the config file would look like so many arguments.